backend/amd64/emit.mlp

```diff
@@ -1440,13 +1443,13 @@ let emit_probe_notes0 () =
       D.qword (const 0)
     end;
     D.qword (ConstLabel semaphore_label);
-    D.bytes "ocaml.1\000";
-    D.bytes (probe_name ^ "\000");
+    D.bytes "ocaml.2\000";
```

Comment:
(Is this something a user might want to query for some
reason? In other words, should this be moved to _e.g._
`Config` and be printed when the compiler is invoked
with `-config`?)

Reply:
Yes, it could be useful, thanks for the suggestion! Probes already have a variable in the Config, I can change it from boolean to probes_version option or something. I'll make a separate PR for it.